src/index.html

```diff
@@ -13,6 +13,38 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the class name 'calendar--start-day-sun' should be formatted in lowercase with hyphens as separators. Also, make sure that the class name reflects its purpose and follows the BEM methodology accurately.